packages/react/src/SelectPanel/SelectPanel.module.css

```diff
@@ -31,3 +36,45 @@
   height: inherit;
   max-height: inherit;
 }
+
+.Message {
+  display: flex;
+  height: 100%;
+  min-height: min(calc(var(--max-height) - 150px), 324px); /* maxHeight of dialog - (header & footer) */
+  padding-top: var(--base-size-24);
```

Comment:
Is there a reason for no x padding here? I think expected would be `padding: var(--base-size-24, 1.5rem);`
<img width="436" alt="Screenshot 2025-03-10 at 1 33 50 PM" src="https://github.com/user-attachments/assets/42235e01-fbdd-4f16-be6a-40c92ad146cb" />

Reply:
done in [5b2f4fe](https://github.com/primer/react/pull/5142/commits/5b2f4fe82faccd84f29f99249aa455d2144fed20)!